dbms/src/main/java/org/polypheny/db/processing/AbstractQueryProcessor.java

```diff
@@ -169,11 +172,17 @@ public PolyphenyDbSignature prepareQuery( RelRoot logicalRoot ) {
 
     @Override
     public PolyphenyDbSignature prepareQuery( RelRoot logicalRoot, RelDataType parameterRowType, boolean isRouted ) {
```

Comment:
Something is strange here...

Reply:
Resolved, if doesSubstituteOrderBy is not specified it always uses false.